pkg/cloudprovider/types.go

```diff
@@ -66,3 +67,9 @@ type Options struct {
 	Client    client.Client
 	ClientSet *kubernetes.Clientset
 }
+
+type Instance struct {
```

Comment:
Can we put this type in the packing package? This types package is limited to the cloud provider APIs (capacty.Create(), capacity...)

Reply:
👍  done!

Reply:
I can go back after this PR and merge NodeCapacity and Instance.